BackofficeBundle/Resources/config/field_type.yml

```diff
@@ -129,6 +129,18 @@ parameters:
                     default_value: choice
                 input:
                     default_value: array
+        document:
```

Comment:
document isn't a little wide ?

I think that it should be content or content_document